### PR TITLE
MAM-3898-nav-cleanup-feed-quick-jump-menu

### PR DESCRIPTION
**Pull request by bnolens**
- long press on home nav bar item
- tutorial for home bar item long-press

[MAM-3898 : Nav Cleanup - Feed quick jump menu](https://linear.app/theblvd/issue/MAM-3898/nav-cleanup-feed-quick-jump-menu)

### CODE REVIEW

File: Mammoth/Screens/TutorialOverlay/Types.swift [hunk 1]
```diff
@@ -0,0 +1,18 @@
+//
+//  Types.swift
```
Comment (rileyhCode) [inline Mammoth/Screens/TutorialOverlay/Types.swift:2]: Can we call this file ArrowAlignment?

**Review by rileyhCode — CHANGES_REQUESTED** [review]
A couple of items to review. Also: do you want to make iPad changes here too, or make a new ticket?

**Comment by bnolens** [comment]
> A couple of items to review. Also: do you want to make iPad changes here too, or make a new ticket?

Good point @rileyhCode. I asked Justin how this should look like on iPad. Waiting on a reply. Not sure the same logic applies to iPad. I'd prefer to create a separate PR for the iPad work.

**Comment by bnolens** [comment]
> > A couple of items to review. Also: do you want to make iPad changes here too, or make a new ticket?
> 
> Good point @rileyhCode. I asked Justin how this should look like on iPad. Waiting on a reply. Not sure the same logic applies to iPad. I'd prefer to create a separate PR for the iPad work.

iPad ticket: https://linear.app/theblvd/issue/MAM-3911/nav-cleanup-ipad-side-column-long-press-actions

**Comment by bnolens** [comment]
> > > A couple of items to review. Also: do you want to make iPad changes here too, or make a new ticket?
> > 
> > 
> > Good point @rileyhCode. I asked Justin how this should look like on iPad. Waiting on a reply. Not sure the same logic applies to iPad. I'd prefer to create a separate PR for the iPad work.
> 
> iPad ticket: https://linear.app/theblvd/issue/MAM-3911/nav-cleanup-ipad-side-column-long-press-actions

Justin says: "... we can do without on iPad."

**Review by rileyhCode — APPROVED** [review]
LGTM

**Pull request merged**